spec/readme.md

```diff
@@ -0,0 +1,15 @@
+
+
+for i in {0..9}; do
```

Comment:
This looks incomplete.

Reply:
Done. May need more next time we add a project - may have missed some details.